File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = s.split(" ")

    for i in range(len(answer)):
        answer[i] = list(answer[i])

    for x in range(len(answer)):
        for y in range(len(answer[x])):
            if y % 2 == 0:
                answer[x][y] = answer[x][y].upper()
            if y % 2 == 1:
                answer[x][y] = answer[x][y].lower()
    
    answer1 = ""
    for row in range(len(answer)):
        if row == 0:
            answer1 += "".join(answer[row])
        else:
            answer1 = answer1 + " " + "".join(answer[row])

    return answer1

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>
#include <cctype>

using namespace std;

string solution(string s) {
    string answer = "";
    int check = 0;
    for (char c : s) {
        if (c != ' ') {
            if (check % 2 == 0) {
                c = toupper(c);
                answer += c;
                check += 1;
            }
            else if  (check % 2 == 1) {
                c = tolower(c);
                answer += c;
                check += 1;    
            }
        }
        else if (c == ' ') {
            answer += c;
            check = 0;
        }
    }
    return answer;
}